Kitodo-DataManagement/src/main/java/org/kitodo/data/database/beans/Process.java

```diff
@@ -17,7 +17,9 @@
 import java.util.List;
 import java.util.Map;
 import java.util.Objects;
+import java.util.stream.Collectors;
 
+import javax.el.PropertyNotWritableException;
```

Comment:
Backport comment from #6085 : Instead of using this exception from the javax namespace an exception class from the Kitodo namespace should be used. Maybe this new exception class is extending this class and if so the needed javax -> jakarta change for this exception class in the near future would only happen at one file instead of many places / files.

I'm writing this only on this place but should be changed on all places.